calendarsubscription/calendarsubscription.go

```diff
@@ -34,19 +34,23 @@ type iCalOptions struct {
 	Shifts          []oncall.Shift `json:"s,omitempty"`
 	ReminderMinutes []int          `json:"r,omitempty"`
 	Version         string         `json:"v,omitempty"`
+	Generated       time.Time      `json:"g,omitempty"`
```

Comment:
Can this be `GeneratedAt`? Also, instead of `iCalOptions` maybe this should be `iCalRenderData` or something like that as it's specific and only used by the template.